openbas-api/src/main/java/io/openbas/rest/atomic_testing/AtomicTestingApi.java

```diff
@@ -74,9 +67,22 @@ public void deleteAtomicTesting(@PathVariable @NotBlank final String injectId) {
     atomicTestingService.deleteAtomicTesting(injectId);
   }
 
-  @GetMapping("/try/{injectId}")
-  public Inject tryAtomicTesting(@PathVariable String injectId) {
-    return atomicTestingService.tryInject(injectId);
+  @PostMapping("/{atomicTestingId}/duplicate")
+  public InjectResultOverviewOutput duplicateAtomicTesting(
+      @PathVariable @NotBlank final String atomicTestingId) {
+    return atomicTestingService.duplicate(atomicTestingId);
+  }
+
+  @PostMapping("/{atomicTestingId}/launch")
+  public InjectResultOverviewOutput launchAtomicTesting(
+      @PathVariable @NotBlank final String atomicTestingId) {
+    return atomicTestingService.launch(atomicTestingId);
+  }
+
+  @PostMapping("/{atomicTestingId}/relaunch")
```

Comment:
Could you add a Tracing annotations on these endpoints (you can have a look at what's already in ExerciseApi.java for an example) ?